frontend/app/components/wp-create/wp-create.controller.ts

```diff
@@ -92,6 +102,12 @@ export class WorkPackageCreateController {
       .catch(error => this.wpNotificationsService.handleErrorResponse(error));
   }
 
+  protected $inject(...args:string[]) {
```

Comment:
Will not work due to WPCopyController being a angular controller subclass, and injectorBridge would only inject the dependencies of the subclass due to the `Constructor.$inject = [..]` notation.

Let's think of some util module that works in all cases and replace all other methods with it in a following PR. Could you think about a method that supports all these cases?